src/styles/_variables.scss

```diff
@@ -0,0 +1,5 @@
+$primary: #F95A57;
+$gray: #797979;
+$lightgray: #D2D2D2;
+
+$roboto: 'Roboto', sans-serif;
```

Comment:
Is there any way we could use system fonts and fall back to Roboto (or vice versa)? Just wondering if there's a way we could get the San Francisco fonts displayed when users are on Apple devices...